changelog.d/14632.misc

```diff
@@ -0,0 +1 @@
+Reject invalid read receipt requests with empty room or event IDs. Contributed by Nick @ Beeper (@fizzadar).
```

Comment:
I'm half tempted to say this is a bugfix, but 🤷

Reply:
Ah good spot, agreed! https://github.com/matrix-org/synapse/pull/14632/commits/fbb7b16148fcc56ac55b6ccd53bcc857987611cd